src/wasm.h

```diff
@@ -91,6 +91,11 @@ struct Name : public cashew::IString {
 // An index in a wasm module
 typedef uint32_t Index;
 
+inline Index toIndex(size_t index, size_t max) {
```

Comment:
Let's do just one parameter for now. For better checks, I think we might find a better option, like a method on the function `func->toLocalIndex(i)`, for example, but let's leave that discussion for later.